search-services/1.3/using/index.md

```diff
@@ -286,16 +286,21 @@ You can narrow your search results by excluding words with the `NOT` syntax.
 
 Single terms, phrases, and so on can be combined using "`NOT`" in upper, lower, or mixed case, or prefixed with "`!`" or "`-`".
 
-These queries search for nodes that contain the terms `yellow` in any content.
+These queries search for nodes that contain the term "yellow" but does not contain the term "banana" in the content.
```

Comment:
Minor correction needed. Change: 

"These queries search for nodes that contain the term "yellow" but do**es** not contain the term "banana" in the content."
To
"These queries search for nodes that contain the term "yellow" but do not contain the term "banana" in the content."

Swap "does" for "do". This is better English. Please apply to other similar lines on this page.

Reply:
Good catch! Thanks for the review.